setup.py

```diff
@@ -325,6 +325,7 @@ def check_file(f):
         cmake_cache_vars = defaultdict(lambda: None, cmake.get_cmake_cache_variables())
         f.write("cuda = {}\n".format(repr(cmake_cache_vars['CUDA_VERSION'])))
         f.write("git_version = {}\n".format(repr(sha)))
+        f.write("hip_version = {}\n".format(repr(cmake_cache_vars['HIP_VERSION'])))
```

Comment:
To mimic cuda, maybe this should be `hip` instead of `hip_version`?

Reply:
Is it going to be `None` in a non-ROCm build? (which is good)

Reply:
I'm renaming it to hip. And yes, in a non-ROCm build HIP_VERSION isn't in the CMakeCache.txt. Then setup.py creates the default dict with a default of None.

Reply:
Thanks!